articles/overview/overview.md

```diff
@@ -0,0 +1,84 @@
+
+The Microsoft Quantum Development Kit (QDK) is a set of open-source tools designed to help developers learn quantum algorithms and write quantum programs. Quantum computing holds the promise to solve some of our planet's biggest challenges - in the areas of environment, agriculture, health, energy, climate, materials science, and others we haven't encountered yet.  
+
+For some of these problems, even our most powerful computers run into problems. While quantum technology is just beginning to impact the computing world, it could be far-reaching and change the way we think about computing.
+
+## What is quantum computing?
+
+In modern usage, the word quantum means the smallest possible discrete unit of any physical property, usually referring to properties of atomic or sub-atomic particles. Quantum computers use actual quantum particles, artificial atoms, or collective properties of quantum particles as processing units, and are large, complex, and expensive devices.
+
+Harnessing the unique behavior of quantum physics and applying it to computing, quantum computers introduce new concepts to traditional programming methods, making use of quantum physics behaviors such as superposition, entanglement, and quantum interference.
+
+## What can a quantum computer do?
+
+A quantum computer isn't a supercomputer that can do everything faster, but there are a few areas where quantum computers do exceptionally well.
+
+- [Quantum simulation](xref:microsoft.quantum.overview.introduction#quantum-simulation)
+- [Cryptography](xref:microsoft.quantum.overview.introduction#cryptography-and-shors-algorithm)
+- [Search](xref:microsoft.quantum.overview.introduction#search-and-grovers-algorithm)
+- [Optimization](xref:microsoft.quantum.overview.introduction#quantum-inspired-computing-and-optimization)
+- [Machine learning](xref:microsoft.quantum.overview.introduction#quantum-machine-learning)
+
+## Quantum simulation
+
+Since quantum computers use quantum phenomena in computation, they are well-suited for modeling other quantum systems. Photosynthesis, superconductivity, and complex molecular formations are examples of quantum mechanisms that quantum programs can simulate.
+
+## Cryptography and Shor’s algorithm
+
+In 1994, Peter Shor showed that a scalable quantum computer could break widely used encryption techniques such as the RSA algorithm. Classical cryptography relies on the intractability of problems such as integer factorization or discrete logarithms, many of which can be solved more efficiently using quantum computers.
+
+## Search and Grover’s algorithm
+
+In 1996, Lov Grover developed a quantum algorithm that dramatically sped up the solution to unstructured data searches, running the search in fewer steps than any classical algorithm could.
+
+> [!NOTE]
+> Ready to try it out? Jump to the Q# tutorial [Implement Grover's search algorithm in Q#](xref:microsoft.quantum.quickstarts.search).
+
+## Quantum-inspired computing and optimization
```

Comment:
Similarly here, this looks like good content, but feels slightly out of place under headings about what quantum computers can do.